cli/EAP7-943_Security_Auth_Commands.adoc

```diff
@@ -0,0 +1,241 @@
+= Title
+:author:            Jean-François Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Extend the existing CLI security command with a set of actions to configure 
+elytron authentication.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-3672
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-943
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
```

Comment:
Fill in your info here btw

Reply:
Which info do you prefer? Btw, mail and author is filled by properties, see the output: https://github.com/jfdenise/wildfly-proposals/blob/master/cli/EAP7-943_Security_Auth_Commands.adoc

Reply:
Ah, my bad! Cool then

Reply:
@mchoma , for SASL, the order is important, the first mechanism is sent to the client.
For http all mechanisms are sent, so the order is not meaningful.

Reply:
@jfdenise you are right, I haven't realized it.